.goreleaser.yml

```diff
@@ -92,6 +95,15 @@ archives:
     builds:
       - cluster-api-runtime-extensions-nutanix
 
+dockers:
```

Comment:
why we decided to not use `ko` to build the image? i.e. building in similar way as CAREN docker image?

Reply:
ko is a tool used for go binaries. i don't think it would be applicable here.

Reply:
we create CAREN image using `kos` goreleaser plugin. https://github.com/nutanix-cloud-native/cluster-api-runtime-extensions-nutanix/blob/main/.goreleaser.yml#L95 
so i thought it would be consistent to use the same for building chart-repo image

Reply:
this is not a go binary so i dont think it works here.